docs/stginga/ipynb.rst

```diff
@@ -3,6 +3,12 @@
 Running stginga with Jupyter/IPython Notebooks
 ==============================================
 
+.. warning::
```

Comment:
@eteq , is this warning enough or do you want one inside the actual notebook?

Reply:
I think this is sufficient.